install_config/router/f5_router.adoc

```diff
@@ -178,14 +178,26 @@ partition.
 See the
 link:https://support.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/tmos_management_guide_10_0_0/tmos_partitions.html[*F5 BIG-IP®* documentation] for more information about administrative partitions.
 
-Use the `--external-host-partition-path` flag when
-xref:deploying-the-f5-router[deploying the F5 router] to specify a partition
-path:
+To configure your {product-title} for partition paths:
 
+. Optionally, perform some cleaning steps:
+.. Ensure F5 is configured to be able to switch to the */Common* and */Custom* paths.
++ 
+.. Delete the static FDB of `vxlan5000`. See
+the
+link:https://support.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/tmos-implementations-12-0-0/9.html[*F5
+BIG-IP®* documentation] for more information.
+
+. xref:f5-configuring-the-virtual-servcers[Configure a virtual server] for the
```

Comment:
The attribute name `f5-configuring-the-virtual-servcers` can be corrected to `f5-configuring-the-virtual-servers`. There are no other references in the project for this attribute.